mdpow/tests/test_fep_script.py

```diff
@@ -0,0 +1,36 @@
+import tempdir as td
+import os
+import manifest
+from mdpow.equil import Simulation
+
+from mdpow.run import fep_simulation
+from mdpow.config import get_configuration
+
+class TestFEPScript(object):
+    
+    def setup(self):
+        self.tmpdir = td.TempDir()
```

Comment:
Test directories should autodestruct after tests are done. Are they? Check `tmpdir`, it can do this kind of stuff.